src/components/App/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { memo, useState, useContext } from 'react';
+import { AuthProvider } from '@contexts/AuthContext';
+import { ThemeProvider } from 'styled-components';
+import { HomeContext, HomeProvider, TABS } from '@contexts/HomeContext';
+import { AuthContext } from '@contexts/AuthContext';
+import SignUp from '@components/SignUp';
+import NotFound from '@components/NotFound';
+import Login from '@components/Login';
+import Home from '@components/Home';
+import NavBar from '@components/NavBar';
+import Profile from '@components/Profile';
+import styled from 'styled-components';
+import {
+  BrowserRouter,
+  Switch,
+  Route,
+} from "react-router-dom";
+import ERRORS from '@errors';
+import history from './history';
+
+const Router = styled(BrowserRouter)`
```

Comment:
Normally you wouldn't style a router. I find a bit weird to apply styles to a component that you're using only to implement core functionalities